### PR TITLE
*: restructure and merge go-sqlsmith and pocket submodule into pkg

### PR DESCRIPTION
**Pull request by mahjonp**
Signed-off-by: mahjonp <junpeng.man@gmail.com>

### What problem does this PR solve? 

There exist three submodule in this repo, so it's unfriendly to maintain dependencies among them.

### What is changed and how does it work?

I restructure go-sqlsmith and pocket, and move them into pkg dir.

test-infra submodule also need to be refined, but we plan to do it after #25 and #24 .

### Check List 

Tests 

 - Unit test
 - Manual test (add detailed scripts or steps below)

Code changes

 - Has Go code change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: db/tidb/sequential_test.go [hunk 1]
```diff
@@ -1,45 +1,46 @@
 package tidb
```
Comment (mapleFU) [inline db/tidb/sequential_test.go]: So where is this testing?

Comment (mahjonp) [reply db/tidb/sequential_test.go]: some workloads aren't used at present, so I commented these code.

**Comment by mahjonp** [comment]
PTAL @you06 @mapleFU

File: Makefile [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+GOTEST=GO15VENDOREXPERIMENT="1" CGO_ENABLED=1 GO111MODULE=on go test # go race detector requires cgo
+VERSION   := $(if $(VERSION),$(VERSION),latest)
+
+PACKAGES := go list ./...| grep -vE 'vendor'
+PACKAGE_DIRECTORIES := $(PACKAGES) | sed 's|github.com/pingcap/tipocket/||'
+
+FILES     := $$(find . -name "*.go" | grep -vE "vendor")
+GOFILTER  := grep -vE 'vendor|render.Delims|bindata_assetfs|testutil|\.pb\.go'
+GOCHECKER := $(GOFILTER) | awk '{ print } END { if (NR > 0) { exit 1 } }'
+GOLINT    := go list ./... | grep -vE 'vendor' | xargs -L1 -I {} golint {} 2>&1 | $(GOCHECKER)
```
Comment (mapleFU) [inline Makefile]: Will this be used ?

Comment (mahjonp) [reply Makefile]: removed

**Pull request merged**